Library/Homebrew/keg_relocate.rb

```diff
@@ -87,16 +87,18 @@ def change_rpath(file, new_prefix)
     end
     return unless patchelf.installed?
     cmd = "#{patchelf.bin}/patchelf --set-rpath #{new_prefix}/lib"
-    (old_i, old_p) = `#{patchelf.bin}/patchelf --print-rpath --print-interpreter #{file}`.split("\n")
+    old_p = `#{patchelf.bin}/patchelf --print-rpath #{file}`.split("\n")
+    old_i = ""
     raise ErrorDuringExecution.new(cmd) unless $?.success?
     if file.mach_o_executable?
+      old_i = `#{patchelf.bin}/patchelf --print-interpreter #{file}`.split("\n")
       interpreter = new_prefix == PREFIX_PLACEHOLDER ?
         "/lib64/ld-linux-x86-64.so.2" :
         HOMEBREW_PREFIX/"lib/ld.so"
       cmd << " --set-interpreter #{interpreter}" unless old_i == interpreter
     end
     cmd << " #{file}"
-    if old_p == "#{new_prefix}/lib" && old_i == interpreter
+    if old_p == "#{new_prefix}/lib" && old_i == interpreter.to_s
       puts "Skipping relocation of #{file} (RPATH already set)" if ARGV.debug?
```

Comment:
Revert this change. With the above two changes, both `old_i` and `interpreter` will be `nil` for a library.